bootstrap/lib/util.js

```diff
@@ -8,6 +8,15 @@ const parseBool = (value) => {
   return (_val === 'true' || _val === 'false') && _val === 'true'
 }
 
+// parse incoming query string, strings into Numbers where possible
+const parseQueryString = (query) => {
```

Comment:
As query parameters are already parsed at this point, and as this fn does something else, consider naming it something more specific. For example:

```suggestion
const toObjectWithNumbers = (query) => {
```